packages/plugin-ext/src/common/plugin-api-rpc.ts

```diff
@@ -1255,6 +1255,8 @@ export interface TaskDto {
     // Provide a more specific type when necessary (see ProblemMatcherContribution)
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     problemMatcher?: any;
+    group?: string,
```

Comment:
minor:

```suggestion
    group?: string;
```

Reply:
Hi Vince !, 
I missed running lint to detect those errors.

A new patch set was submitted addressing your comments and a fix that was braking tests.
Thanks !
/Alvaro